eq-author/src/App/QuestionnairesPage/QuestionnairesTable/index.js

```diff
@@ -55,6 +58,21 @@ export class UnconnectedQuestionnairesTable extends React.PureComponent {
 
   state = {
     focusedId: null,
+    showDeleteQuestionnaireDialog: false,
+    deleteQuestionnaire: {},
```

Comment:
Could we default this to `null` or `undefined` so that its obvious its not there. `{}` is truthy

Reply:
Changed to null with other corresponding changes to cope with null value